.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
I would prefer if we add `.vscode/` to the gitignore and keep these as local settings.